test/python/picongpu/quick/pypicongpu/species/constant/groundstateionization.py

```diff
@@ -0,0 +1,142 @@
+"""
+This file is part of PIConGPU.
+Copyright 2021-2023 PIConGPU contributors
```

Comment:
minor: in most other places you updated the year

Reply:
fixed